package.json

```diff
@@ -44,6 +44,7 @@
     "@fortawesome/free-solid-svg-icons": "5.11.2",
     "@fortawesome/react-fontawesome": "0.1.9",
     "babel-polyfill": "6.26.0",
+    "classnames": "^2.2.6",
```

Comment:
> Do we use this library in other frontend-apps?

Yes, I think it is used in almost all other frontend app, and in edx-platform itself. While I had prior experience with it, I did look into it before suggesting it here. 

Even if we'd like to avoid another dependency (npm dependencies can quickly get unwieldy), I think a helper for this is a pretty good thing to include in the `frontend-platform`.

Reply:
Unless @davidjoy protests, I'm fine with this library then, thanks for checking.

If we could though, still refactor its usage to break up the long line(s) so the conditional logic isn't hidden off-screen?

Reply:
Yes, we use classnames in many apps and have found it really useful. 👍   It shorthands a lot of complicated logic very nicely.